model/project_selector.go

```diff
@@ -275,6 +275,38 @@ func (t *taskSelectorEvaluator) evalSelector(s Selector) ([]string, error) {
 	return results, nil
 }
 
+// Display task selector
+type displayTaskSelectorEvaluator struct {
+	tagEval *tagSelectorEvaluator
+}
+
+func NewDisplayTaskSelectorEvaluator(bv BuildVariant, tasks []parserTask) *displayTaskSelectorEvaluator {
+	var selectees []tagged
+	for _, t := range bv.Tasks {
+		selectees = append(selectees, &parserTask{Name: t.Name, Tags: getTags(tasks, t.Name)})
+	}
+	return &displayTaskSelectorEvaluator{
+		tagEval: newTagSelectorEvaluator(selectees),
+	}
```

Comment:
public functions shouldn't return private types. 

having said that, I don't particularly see why there needs to be a separate type and/or why we couldn't get away with using composition (e.g. anonymous fields)

Reply:
removed the separate type